packages/mobile/src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx

```diff
@@ -200,11 +202,19 @@ export const ChallengeRewardsDrawer = ({
   const statusText = hasCompleted
     ? messages.complete
     : isInProgress
-    ? `${currentStep}/${stepCount} ${progressLabel}`
+    ? fillString(
```

Comment:
FYI @raymondjacobson I think this changed to use `fillString` instead of just being the label. (Originally it was just "Invites Accepted" before your change - now it's "%0/%1 Invites Accepted")

Reply:
Oh shoot, thank you for finding this